src/Confluent.Kafka/Consumer.cs

```diff
@@ -316,13 +319,13 @@ public void Poll()
         public event EventHandler<Message<TKey, TValue>> OnMessage;
 
         /// <summary>
-        ///     Returns the current partition assignment as set by Assign.
+        ///     The current partition assignment as set by Assign.
```

Comment:
Returns is not the best terminology for a property... but now you're made me think about it, properties are usually documented as "Gets ..." or "Gets and Sets ..." - I'll go through and change a lot of them over.